### PR TITLE
🐛(frontend) redirect on unauthorized pages

### PR DESCRIPTION
**Pull request by rlecellier**
Teacher dashboard route should redirect to learner dashbaord route when
the user isn't authorized.

### CODE REVIEW

File: src/frontend/jest/setup.ts [hunk 1]
```diff
@@ -25,15 +25,30 @@ RESET_MODULE_EXCEPTIONS.forEach((moduleName) => {
   });
```
Comment (jbpenrath) [inline src/frontend/jest/setup.ts:25]: This commit should be drop from this PR

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
```
Comment (jbpenrath) [inline CHANGELOG.md:15]: In the commit message :
`s/dashbaord/dashboard/`

File: src/frontend/js/hooks/useResources/useResourcesRoot.ts [hunk 3]
```diff
@@ -165,6 +165,9 @@ export const useResourcesRoot = <
     items: TData[];
```
Comment (jbpenrath) [inline src/frontend/js/hooks/useResources/useResourcesRoot.ts:165]: This is out of scope of this PR

File: CHANGELOG.md [hunk 4]
```diff
@@ -15,6 +15,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Fixed
 
+- Teacher dashboard route now redirect to learner dashbaord route when
+  the user isn't authorized.
```
Comment (jbpenrath) [inline CHANGELOG.md:19]: IMO this is a change not a fix

File: src/frontend/jest/setup.ts (hunk 1 above)
Comment (rlecellier) [reply src/frontend/jest/setup.ts:25]: it is, it's here: #2376

File: CHANGELOG.md [hunk 5]
```diff
@@ -10,6 +10,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Added
 
+- Teacher dashboard route now redirect to learner dashbaord route when
```
Comment (jbpenrath) [inline CHANGELOG.md:13]:
```suggestion
- Teacher dashboard route now redirect to learner dashboard route when
```

**Review by jbpenrath — APPROVED** [review]
